CHANGELOG.md

```diff
@@ -5,6 +5,68 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.3.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.3.0) - 2023-10-16
+
+This release adds RPC methods for the "Spend before Sync" light wallet feature,
+and fixes performance issues and bugs in the mining solution rate RPCs. Progress
+bars can now be enabled using a config, please help us test them!
+
+It contains the following updates:
+ 
+### User Testing: Progress Bars
+ 
+Zebra has progress bars! When progress bars are enabled, you can see Zebra's blocks,
+transactions, and peer connections in your terminal. We're asking Zebra users to test this
+feature, and give us [feedback on the forums](https://forum.zcashcommunity.com/t/zebra-progress-bars/44485).
+
+To show progress bars while running Zebra, add these lines to your `zebrad.toml`:
+```toml
+[tracing]
+progress_bar = "summary"
+```
+
+For more details, including a known issue with time estimates,
+read our [progress bars blog post](https://zfnd.org/experimental-zebra-progress-bars/).
+
+### Security
+
+- Fix database concurrency bugs that could have led to panics or incorrect history tree data (#7590, #7663)
+
+### Added
+
+- Zebra's progress bars can now be enabled using a `zebrad.toml` config (#7615)
+- Add missing elasticsearch flag feature to lib docs (#7568)
+- Add missing Docker variables and examples (#7552)
+
+#### Spend before Sync Support
+
+- Add state requests and support code for the `z_getsubtreesbyindex` RPC (#7408, #7734)
+- Implement the `z_getsubtreesbyindex` RPC (#7436)
+- Test the `z_getsubtreesbyindex` RPC (#7515, #7521, #7566, #7514, #7628)
+- Format subtree roots in little-endian order (#7466)
+- Add note subtree indexes for new and existing blocks (#7437)
+- Upgrade subtrees from the tip backwards, for compatibility with wallet syncing (#7531)
+- Handle a subtree comparison edge case correctly (#7587)
+
+### Changed
+
+- Return errors instead of panicking in methods for Heights (#7591)
+- Update tests for compatibility with the ECC's `lightwalletd` fork (#7349)
+- Put chain data into an inner struct (#7608)
+
+### Fixed
+
+- Refactor docs for feature flags (#7567)
+- Match zcashd's getblockchaininfo capitalisation for NU5 (#7454)
+- Populate subtree fields in the `NoteCommitmentTrees` struct (#7636)
```

Comment:
Optional Nitpick: users can't see this change.

Reply:
Done in [8c83495](https://github.com/ZcashFoundation/zebra/pull/7610/commits/8c83495daae272013004fdba550fa739bbd4b191).